staging/src/k8s.io/legacy-cloud-providers/azure/azure_cache.go

```diff
@@ -26,6 +26,20 @@ import (
 	"k8s.io/client-go/tools/cache"
 )
 
+// cacheReadType defines the read type for cache data
+type cacheReadType int
+
+const (
+	// cachedData returns data from cache if cache entry not expired
+	// if cache entry expired, then it will refetch the data using getter
+	// save the entry in cache and then return
+	cachedData cacheReadType = iota
```

Comment:
could we add prefixes for those consts, e.g. `cacheReadyTypeDefault` and `cacheReadTypeUnsafe`

Reply:
that's a good point. I'll add the prefixes.

Reply:
Done!